packages/cli/lib/prompts/selectPublicAppPrompt.js

```diff
@@ -64,7 +70,8 @@ const selectPublicAppPrompt = async ({
       }),
       type: 'list',
       choices: publicApps.map(app => {
-        if (migrateApp && app.preventProjectMigrations) {
+        const { preventProjectMigrations, listingInfo } = app;
+        if (migrateApp && preventProjectMigrations && listingInfo) {
```

Comment:
Do we need the `migrateApp` argument because you're also using this prompt for the clone app flow?

Reply:
Yes, exactly.

Reply:
Would you mind renaming it to something along the lines of `isMigratingApp`? Just so the logic is a little easier to follow

Reply:
No prob!

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1114/commits/575fa8706bfc21cc7c13a98b76aee15e4fb411f3